WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -30,6 +30,7 @@ protocol CardPresentPaymentsOnboardingUseCaseProtocol {
 final class CardPresentPaymentsOnboardingUseCase: CardPresentPaymentsOnboardingUseCaseProtocol, ObservableObject {
     let storageManager: StorageManagerType
     let stores: StoresManager
+    var stripeGatewayIPPEnabled: Bool?
```

Comment:
nit: could this be private?
```suggestion
    private var stripeGatewayIPPEnabled: Bool?
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5734/commits/60739088b75ba33d4cf69b4aaef15c576e5f8e56